lib/stats.dart

```diff
@@ -156,11 +163,16 @@ class Stats {
     totalCoins += coins;
   }
 
-  static List<T> normalize<T>(List<T> scores) => scores.sublist(0, MAX_SCORES.clamp(0, scores.length));
+  static List<T> normalize<T>(List<T> scores) =>
+      scores.sublist(0, MAX_SCORES.clamp(0, scores.length));
 
   static Stats merge(Stats stats1, Stats stats2) {
     return new Stats()
-      ..scores = normalize((new Set()..addAll(stats1.scores)..addAll(stats2.scores)).toList().cast<Score>())
+      ..scores = normalize((new Set()
```

Comment:
remove the new

Reply:
I will add linter in a followup and remove all news and other old artifacts in a single scoop